File: Java/Advanced Conditional Statements/Lab/03. Animal Type/AnimalType.java
import java.util.Scanner;

public class AnimalType {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        String animalName = input.nextLine();

        switch(animalName){
            case "dog":
                System.out.println("mammal");
                break;
            case "crocodile":
            case "tortoise":
            case "snake":
                System.out.println("reptile");
                break;
            default:
                System.out.println("unknown");
        }
    }
}

File: Python/Basics/Conditional Statements Advanced/Lab/Animal Type.py
# dog -> mammal
# crocodile, tortoise, snake -> reptile
# others -> unknown

animalType = input()

if animalType == "dog":
    print("mammal")
elif animalType == "crocodile" or animalType == "tortoise" or animalType == "snake":
    print("reptile")
else:
    print("unknown")